mage_ai/frontend/components/suggestions/SuggestionRow.tsx

```diff
@@ -72,6 +72,8 @@ const SuggestionRow = ({
     action_options: actionOptions,
   } = action_payload;
 
+  useEffect(() => setActionPayload(action_payload), [action_payload]);
```

Comment:
@shrey-mage does it matter that this comes after `const [actionPayload, setActionPayload] = useState<ActionPayloadType>(action_payload);`?

Reply:
not sure if that will cause any issues